docs/qa/test_engineering

```diff
@@ -0,0 +1,44 @@
+Normandy Test Engineering
+=========================
+This is the repository for all information related to test engineering for the Normandy project.  This will outline our resources and test process for releases.
+
+The primary focus of testing will be on the Firefox client and the API server. The Normandy Firefox client follows the release process and train schedule for the Firefox browser. The Normandy server is tested using the API server and recipes.
+
+The Delivery Console is not currently being tested, as it’s a much lower priority. 
+
+The test strategy incorporates manual and automated testing. Different approaches are outlined for each area. 
+
+Normandy Server
+--------------
```

Comment:
RST is picky and requires that the underline here is the exact same length as the line it is underlining.

Reply:
fixed